components/ledger/internal/services/command/finalize_transaction_persistence.go

```
@@ -0,0 +1,579 @@
+func replayJSONIdentityMatches(raw string, expected *transaction.Transaction) bool {
+	existing := &transaction.Transaction{}
+
+	return json.Unmarshal([]byte(raw), existing) == nil && replayIdentityMatches(existing, expected)
+}
+
+func (uc *UseCase) completeUnownedReplay(
+	ctx context.Context,
+	key, encoded string,
+	replay *transaction.Transaction,
+) error {
+	const replayTTLSeconds time.Duration = 300
```

golangci-lint (2.12.2)

[error] ST1011: var replayTTLSeconds is of type time.Duration; don't use unit-specific suffix "Seconds"

(staticcheck)